languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -35063,7 +35063,15 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TOURS_CRITIQUES" name="Tours critiqués divers (barbarismes, impropriétés, solécismes, etc.)">
-    <rulegroup id="POUR_PAS_QUE" name="familier : pour (ne) pas que...">
+      <rule id= "RENUMERER_OU_REMUNERER" name="rénumérer ou rémunérer">
+          <pattern>
+              <token inflected="yes">rénumérer</token>
```

Comment:
The thing is: our dictionary doesn't know about the very rare verb "rénumérer". So it cannot "inflect" it. You can either put the expected verb forms in there (with plenty of regex) or — which may be easier — we can improve the suggestions given for any typo starting with `rénumér.*`. Remind me about this on Monday :)

Reply:
Here is a solution (powered by Jaume): make the token `<token regexp="yes">rénum.*</token>`. Le message devient `Une erreur d’orthographe semble être détectée.`. La suggestion sera `<suggestion suppress_misspelled="yes"><match no="1" regexp_match="(?iu)num" regexp_replace="mun" case_conversion="preserve"/></suggestion>`.

Please check whether the rule, in this form, works okay and doesn't produce false positives.

Reply:
We will discuss this on Monday :)